rebar.config.erl

```diff
@@ -543,10 +553,11 @@ dialyzer(Config) ->
         end,
 
     AppNames = app_names(),
+    ExcludedApps = excluded_apps(standard),
 
     KnownApps = [Name || Name <- AppsToAnalyse, lists:member(Name, AppNames)],
 
-    AppsToExclude = AppNames -- KnownApps,
+    AppsToExclude = ExcludedApps -- KnownApps ++ AppNames,
```

Comment:
This is effectively the same as: `AppsToExclude = ExcludedApps -- (KnownApps ++ AppNames)`
`KnownApps` is a subset of `AppNames`, I don't understand why the ++ here.

Reply:
did you mean `AppsToExclude = ExcludedApps -- AppNames`?

Reply:
yes.